src/manifolds/HyperbolicHyperboloid.jl

```diff
@@ -239,6 +239,93 @@ function exp!(M::Hyperbolic, q, p, X)
     return copyto!(q, cosh(vn) * p + sinh(vn) / vn * X)
 end
 
+function get_basis(M::Hyperbolic, p, B::DefaultOrthonormalBasis)
+    n = manifold_dimension(M)
+    V = [_hyperbolize(M, p, [i == k ? 1 : 0 for k in 1:n]) for i in 1:n]
+    return CachedBasis(B, gram_schmidt(M, p, V))
+end
+
+function get_basis(M::Hyperbolic, p, B::DiagonalizingOrthonormalBasis)
+    n = manifold_dimension(M)
+    X = B.frame_direction
+    V = [_hyperbolize(M, p, [i == k ? 1 : 0 for k in 1:n]) for i in 1:n]
```

Comment:
```suggestion
    V = [_hyperbolize(M, p, [i == k ? one(eltype(p)) : zero(eltype(p)) for k in 1:n]) for i in 1:n]
```

Reply:
Ah, yes, I have to check for those more carefully.